CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ enable_testing()
 if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
   set(gtest_force_shared_crt ON CACHE BOOL "Make gtest compile the C standard libraries as DLLs, because MSVE requires that." FORCE)
 endif()
-add_subdirectory(googletest)
+add_subdirectory(googletest EXCLUDE_FROM_ALL)
 
```

Comment:
I see that this works and seems to also build tests but I don't understand it.
How comes, googletest is actually build when I run cmake --build ...?

Reply:
This only excludes that directory from installs https://stackoverflow.com/a/64900982/6662425